### PR TITLE
Feature/lib improvement

### PR DESCRIPTION
**Pull request by SabreFoxx-Reloadly**

### CODE REVIEW

**Review by iSeremet-Reloadly — APPROVED** [review]

File: src/lib/components/select/select.component.ts [hunk 1]
```diff
@@ -93,11 +91,12 @@ export class SelectComponent implements OnInit, OnChanges, ControlValueAccessor
         this.list.next(this.options);
     }
 
-    toggleDropdown() {
-        if (this.type == 'input' || this.disabled) return;
-        this.showDropdown = !this.showDropdown;
```
Comment (farouk-reloadly) [inline src/lib/components/select/select.component.ts:98]: Any reason why you removed the early return?

**Review by farouk-reloadly — APPROVED** [review]

File: src/lib/components/select/select.component.ts (hunk 1 above)
Comment (SabreFoxx-Reloadly) [reply src/lib/components/select/select.component.ts:98]: Yes. The toggleDropdown method isn't meant to be called when the type is input. So while debugging, I needed to make the condition clearer textually, and also I noticed something a bit strange earlier, and after I chaned to this, I couldn't reproduce that strange behavior which had to do with toggling the dropDown. Please also look at the changes I made to select.component.html

File: src/lib/containers/card/card.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
-<div class="card is-flex is-flex-direction-column">
-    <div class="is-flex is-justify-content-space-between">
+<div class="rld-card">
```
[the remark this answers is not in the document]
Comment (SabreFoxx-Reloadly) [reply src/lib/containers/card/card.component.html:1]: Okay

[the remark this answers is not in the document]
Comment (SabreFoxx-Reloadly) [reply src/lib/containers/card/card.component.html:1]: But I think this particular one is internal

**Pull request merged**